app/controllers/offences_controller.rb

```diff
@@ -13,21 +13,19 @@ class OffencesController < ApplicationController
   skip_load_and_authorize_resource only: [:index]
 
   def index
-    if permitted_params[:fee_scheme].present?
-      @offences = FeeReform::SearchOffences.call(permitted_params)
-
-      respond_to do |format|
-        format.json do
-          render json: @offences, each_serializer: FeeReform::OffenceSerializer
-        end
-      end
-    else
-      @offences = Offence.in_scheme_nine.where(offence_filter)
+    respond_to do |format|
+      format.json { render json: offences, each_serializer: FeeReform::OffenceSerializer }
```

Comment:
name wise if we are using this serializer for all offences it should not be namespaced to `FeeReform`, which relates to AGFS fee scheme 10+ only